news/2024/2024-03-19-bespoke-music-in-osu.md

```diff
@@ -0,0 +1,259 @@
+> 
+> I feel like the taiko side definitely has larger budgets, and may have more sophisticated processes when it comes to the custom songs planning and producing process, as they had a custom song for every round of the [osu!taiko Malaysian Tournament 2022](/wiki/Tournaments/OTMT/2022), unlike std where we have 1-3 custom songs for the entire tournament
+
+[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/Jerry.png)](https://osu.ppy.sh/users/605973)
+
+> Hello! My name is Jerry (or NZMJerry), and I'm mostly known for my design works in a number of osu! tournaments and original tracks such as the [Global Taiko Showdown](https://gtsosu.com/) (GTS) series, [osu! World Cup 2023](https://osu.ppy.sh/beatmapsets/2095177#taiko/4393066) and [Jaye's One Key Event (JOKE)](https://osu.ppy.sh/community/forums/topics/1829307?n=1). Besides my designing works, I'm also the host of the osu!taiko Malaysian Tournament (OTMT) series as well as several other events ranging from contests to [LANs](https://osu.ppy.sh/home/news/2023-08-22-osu-malaysia-animangaki) for the overall osu! Malaysian community.
+> 
+> After wrapping up the [first edition of the osu!taiko Malaysian Tournament](https://osu.ppy.sh/community/forums/topics/1389964?n=1) (OTMT) series back in 2021, the team didn't want to just do the same thing again for the next year and beyond, so we felt that the most natural progression for the series' growth was to start introducing original songs into the mix.
+> 
+> I truly believe that this, along with the introduction of our mascot character, **IRIS**, helped a lot in weaving a recognizable visual identity for the whole tournament, which significantly enhanced the tournament experience.
+> 
+> ![osu!taiko Malaysia mascot "IRIS"](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/iris.jpg)
+
+Visual arts go hand in hand with music commissions and tournament production, which is itself an interesting topic (that we may cover in the near future!).
+
+> The music production process for our team is rather straightforward. We'll usually only specify the particular genre and difficulty that we're looking for to an artist based on the style that they're comfortable with, along with some example tracks provided. Besides that, the artists are given creative liberties to elevate it however they wish.
+> 
+> I think this process works very well as we also have a general channel for all our composers to mingle and share ideas for each other's works, allowing for more constructive opinions to be provided throughout the process by equally skilled individuals.
+
+---
+
+The Malaysian tournaments collectively have 19 original tracks in total at the moment, but word is on the street that more is on the way!
```

Comment:
when reading this for the first time i felt like i fumbled my words, maybe this reads better?
```suggestion
The Malaysian tournaments collectively have 19 original tracks in total at the moment, but the word on the street is that more is on the way!
```